orderfilter/filter_js.go

```diff
@@ -0,0 +1,46 @@
+// +build js, wasm
+
+package orderfilter
+
+import (
+	"fmt"
+	"strings"
+	"syscall/js"
+
+	"github.com/0xProject/0x-mesh/ethereum"
+)
+
+type Filter struct {
+	validatorLoaded      bool
```

Comment:
I think this field, along with the `checkForValidator` function are no longer necessary because we are setting the validator directly by calling `setSchemaValidator`?

This also brings up the question, why do we need to set the value on `window` at all? Could we simply return the validator from `setSchemaValidator` and store it as a `js.Value` field on `Filter`? If so we should probably rename `setSchemaValidator` to something like `createSchemaValidator` or use a JavaScript constructor directly by calling `(js.Value).New`.

Reply:
Good point 👍